tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-  "extends": "@electron-toolkit/tsconfig/tsconfig.node.json",
+  "extends": "./node_modules/@electron-toolkit/tsconfig/tsconfig.node.json", // Updated path
```

Comment:
The standard JSON specification doesn't have “comments”, and this code is not valid.